docs/aqua-book/changelog.md

```diff
@@ -7,6 +7,13 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+### [0.8.0](https://github.com/fluencelabs/aqua/releases/tag/0.8.0) – November 30, 2022
+* Structure objects can be created right in the code ([example](https://fluence.dev/docs/aqua-book/language/types.md#structures))
+* `module` key word changed to `aqua` ([docs](https://fluence.dev/docs/aqua-book/language/header/header.md#module))
+* Syntax checking has been disabled in generated JS/TS files
```

Comment:
I would love it if you specify who did that checking. Like `Syntax checking by tsc has been disabled in generated JS/TS files`.

Otherwise it is wtf.